lib/cc/engine/analyzers/violation.rb

```diff
@@ -91,19 +112,15 @@ def fingerprint
           digest << "-"
           digest << current_sexp.mass.to_s
           digest << "-"
-          digest << occurrences.to_s
+          digest << occurrences.length.to_s
```

Comment:
> Do you know how we did that in classic?

I don't. And please don't take my comments as definitely accurate.

This is a thorny problem with lots of landmines, and the desired behavior is pretty subtle (for example, is cross-file handling different than same-file?). I think we might need to pick @brynary 's brain about desired behavior, then dig into how the Classic code is implemented.